shell/platform/fuchsia/dart_runner/dart_component_controller_v2.cc

```diff
@@ -295,16 +292,21 @@ bool DartComponentControllerV2::SetUpFromAppSnapshot() {
       return false;
     }
   } else {
+    // TODO(fxb/91200): This code path was broken for over a year and is
+    // probably not used.
     if (!dart_utils::MappedResource::LoadFromNamespace(
             namespace_, data_path_ + "/isolate_snapshot_data.bin",
             isolate_snapshot_data_)) {
       return false;
     }
+    isolate_data = isolate_snapshot_data_.address();
+
     if (!dart_utils::MappedResource::LoadFromNamespace(
             namespace_, data_path_ + "/isolate_snapshot_instructions.bin",
             isolate_snapshot_instructions_, true /* executable */)) {
       return false;
     }
+    isolate_instructions = isolate_snapshot_instructions_.address();
```

Comment:
why not get rid of the code for mapping isolate_snapshot_instructions.bin from this path too?

Reply:
I wasn't certain if the .bin in this code path was coming from our core snapshots or created elsewhere, e.g. in the component build rules in the fuchsia tree. However this code is likely dead so removing it.